app/src/main/java/com/zulip/android/networking/GravatarAsyncFetchTask.java

```diff
@@ -47,7 +48,7 @@ public static URL sizedURL(Context context, String url, float dpSize) {
         float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                 dpSize, r.getDisplayMetrics());
         try {
-            return new URL(url + "&s=" + px);
+            return new URL(UrlHelper.addHost(url)  + "&s=" + px);
```

Comment:
@timabbott Doing this change can mess up things for the GCM for zulipchat servers as well, so now will the servers not use GCM and use push bouncer? 

(I never got to work with GCM so I'm not sure what `sender_avatar_url` string contains!